lerobot/common/policies/act/modeling_act.py

```diff
@@ -38,7 +38,12 @@
 from lerobot.common.policies.normalize import Normalize, Unnormalize
 
 
-class ACTPolicy(nn.Module, PyTorchModelHubMixin):
+class ACTPolicy(nn.Module, PyTorchModelHubMixin, 
+                library_name="lerobot",
+                repo_url="https://github.com/huggingface/lerobot",
+                docs_url="https://github.com/huggingface/lerobot/",
```

Comment:
No need for `docs_url` since it's the same as `repo_url` (same in other classes)

```suggestion
```